pkgs/desktops/plasma-5/3rdparty/kwin/kde-rounded-corners.nix

```
@@ -0,0 +1,49 @@
+{ pkgs, qtbase, qttools, qtx11extras, kconfig, kcoreaddons, ki18n,
+kio, kglobalaccel, kinit, kwin, xorg, libepoxy, lib }:
+
+{
+  kde-rounded-corner = pkgs.stdenv.mkDerivation {
```

Youre exporting a set containing kde-reounded-corner rather than the derivation itself.

```suggestion
stdenv.mkDerivation {
```